src/components/Select/index.tsx

```diff
@@ -63,32 +63,39 @@ type StyledSelectProps = Omit<SelectProps, 'errorMesage' | 'hasError' | 'label'>
   label?: string;
 };
 
-const Select: FunctionComponent<SelectProps> = ({
-  errorMessage,
-  hasError,
-  required,
-  theme: propsTheme,
-  label,
-  hintText,
-  ...props
-}) => {
-  const theme = {
-    ...defaultTheme,
-    ...propsTheme,
-  };
+const Select: FunctionComponent<SelectProps> = forwardRef(
+    (
+      {
+        errorMessage,
+        hasError,
+        required,
+        theme: propsTheme,
+        label,
+        hintText,
+        ...props
+      }: SelectProps,
+      ref?: Ref<HTMLInputElement>
```

Comment:
the type might want to be HTMLSelectElement I'm not sure what it wants, without code assist I'm just guessing tbh

Reply:
Ref<HtmlSelectElement> as agreed